src/components/common/ColonyActionsTable/partials/ActionMobileDescription/ActionMobileDescription.tsx

```diff
@@ -5,25 +5,39 @@ import React, { type FC } from 'react';
 import getActionTitleValues from '~common/ColonyActions/helpers/getActionTitleValues.ts';
 import { useColonyContext } from '~context/ColonyContext/ColonyContext.ts';
 import useShouldDisplayMotionCountdownTime from '~hooks/useShouldDisplayMotionCountdownTime.ts';
-import { MotionState } from '~utils/colonyMotions.ts';
 import { formatText } from '~utils/intl.ts';
+import { useGetExpenditureData } from '~v5/common/ActionSidebar/hooks/useGetExpenditureData.ts';
 import MotionCountDownTimer from '~v5/common/ActionSidebar/partials/Motions/partials/MotionCountDownTimer/index.ts';
-import MotionStateBadge from '~v5/common/Pills/MotionStateBadge/index.ts';
 import TeamBadge from '~v5/common/Pills/TeamBadge/index.ts';
 import MeatBallMenu from '~v5/shared/MeatBallMenu/index.ts';
 
+import ActionBadge from '../ActionBadge/index.ts';
```

Comment:
Please import the file directly, and remove the use of the index barrel file

```suggestion
import ActionBadge from '../ActionBadge/ActionBadge.tsx';
```